docs/source/conf.py

```diff
@@ -206,8 +206,10 @@
 
 # -- Type hints configs ------------------------------------------------------
 
+autodoc_inherit_docstrings = True
```

Comment:
Default is also True, but let's keep it like that

https://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html

Reply:
Yes, we want to have `True` value, but keeping it just in case sphinx change to `False`